### ISSUE TITLE
Revision: update telomerase analysis to assess whether samples with TERTp have high scores

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-analysis#1651]
#### What analysis module should be updated and why?
https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/telomerase-activity-prediction

Reviewer commented that we did not assess whether samples with TERT promoter (TERTp) mutations have high telomerase scores

#### What changes need to be made? Please provide enough detail for another participant to make the update.
Plot telomerase scores and highlight tumors with  _TERT_ promoter mutations C228T and C250T? 

#### What input data should be used? Which data were used in the version being updated?
SNV consensus MAF + hotspot MAF (to ensure we capture all TERTp)
telomerase scores

#### When do you expect the revised analysis will be completed?
this week?

#### Who will complete the updated analysis?
?

**Comment by sjspielman** [comment]
Hi @jharenza I'm going to start having a look here. I'm wondering if you have a reference for the TERTp mutations `C228T` and `C250T` you listed? I found this paper which does specifically list those two (https://aacrjournals.org/mcr/article/14/4/315/132821/Understanding-TERT-Promoter-Mutations-A-Common), but I wonder if we should be looking more generally for any mutations upstream of TERT within a given bp range?
That said I have yet to actually look at the data so I don't know how much of this is actually in our WGS data!

**Comment by jharenza** [comment]
I don't think so - we may not be capturing them, if so (eg - could be a situation of 1 or 2/4 callers) - these are the two hotspot mutations (more in the README [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/hotspots-detection))

**Comment by sjspielman** [comment]
@jharenza Ok, I've poked around a bit. The data here is pretty thin (too thin?) I think. To just get something started, I looked at just _any_ tumors with a mutation in the TERT 5' flank. There are only 15 samples (pooled from both the hotspots and consensus MAF files), and only 12 of those have corresponding stranded RNA samples. I didn't dive into genomic coordinates to check if any of these are _those two_ hotspot mutations. A quick wilcox test between samples with and without mutations shows P=0.119.

Black horizontal line below is the global median.
<img width="717" alt="Screenshot 2023-01-19 at 4 20 07 PM" src="https://user-images.githubusercontent.com/4701111/213563439-a43827a9-a116-4445-b585-8f261439e48c.png">


I can go ahead and file this as a PR but wanted to check first in case this strategy is just not at all the right way to go!
